lib/NfcoreSchema.groovy

```diff
@@ -12,17 +12,24 @@ import groovy.json.JsonSlurper
 import groovy.json.JsonBuilder
 
 class NfcoreSchema {
```

Comment:
`NfCoreSchema`? and in the same way `NfCoreTemplate`... :roll_eyes:

Reply:
I don't know that difference between "C" "c" can be hard to catch, I would rather leave it simple with "Nfcore"

Reply:
Yep.  I think we should just keep it the same as it is now. Still a bit annoyed that we have to use `N` to make Groovy happy :)